lib/codelog/presenters/version_data_presenter.rb

```diff
@@ -0,0 +1,21 @@
+module Codelog
+  module Presenters
+    class VersionDataPresenter
+      def initialize(version_changes_data)
+        @changes = version_changes_data
+      end
+
+      def date
+        @changes['Date'].strftime(Codelog::Config.date_output_format)
+      end
+
+      def version
+        @changes['Version']
+      end
+
+      def modifications
+        @changes.dup.delete_if { |key| key == 'Version' || key == 'Date' }
```

Comment:
It should be better if you add the keys that need to be excluded in a array, and use 
`delete_if { |key| exclusion_array.include?(key) }`